test/fail_compilation/fail196.d

```diff
@@ -10,10 +10,10 @@ fail_compilation/fail196.d(29): Error: found `";\x0a    assert(s == "` when expe
 fail_compilation/fail196.d(30): Error: found `");\x0a\x0a    s = q"` when expecting `;` following statement
 fail_compilation/fail196.d(32): Error: found `{` when expecting `;` following statement
 fail_compilation/fail196.d(32): Error: found `}` when expecting `;` following statement
-fail_compilation/fail196.d(33): Error: found `foo` when expecting `;` following statement
+fail_compilation/fail196.d(33): Error: Found `{` when expecting `;` or `=`, did you mean `";\x0a    assert(s == " foo = xxx`?
 fail_compilation/fail196.d(33): Error: found `}` when expecting `;` following statement
 fail_compilation/fail196.d(35): Error: found `<` when expecting `;` following statement
-fail_compilation/fail196.d(36): Error: found `foo` when expecting `;` following statement
+fail_compilation/fail196.d(36): Error: Found `<` when expecting `;` or `=`, did you mean `xxx >> ";\x0a    assert(s == " foo = xxx`?
```

Comment:
I know the code in this test is nonsense anyway, but perhaps this is a small sign that the suggestion style message should be more restricted to parsed tokens that could make sense in the rewritten form (a shift expression can never be a type).